benchadapt/python/benchadapt/adapters/folly.py

```diff
@@ -72,6 +72,7 @@ def _transform_results(self) -> List[BenchmarkResult]:
                         "unit": "ns",
                         "times": [result[2]],
                         "time_unit": "ns",
+                        "iterations": 1,
```

Comment:
I'm not sure this couldn't be higher—does folly have an idea of replications?—but this is implicit in our current processing of the results

Reply:
Yeah, I alluded to this in my other comment: I don't actually think folly returns # of iterations that ran. Though it _does_ at some point have some number of iterations that it runs. 

We probably want at some point to treat these microbenchmarks which run for a set time period and measures how many iterations it gets through slightly differently than our macro benchmarks where each iteration happens regardless of how long it takes. But not today!

Reply:
`iterations` here is more what archery calls `replications`, not the actual number of iterations run in a period of time (that's in `data` in iterations/s). If we wanted to store raw iterations, we'd need to do it elsewhere, since this field is a single integer, and can't handle a vector of values like you'd get for multiple replications.